server/src/controllers/userControllers.ts

```diff
@@ -140,11 +140,7 @@ export const getUser = async (
   }
 };
 
-export const getAllUsers = async (
-  req: Request,
-  res: Response,
-  next: NextFunction
-) => {
+const getUsersFromDB = async (next: NextFunction) => {
```

Comment:
@whoadood I like that you separated this function out. It will make testing and error handling easier.